workshops/util.py

```diff
@@ -366,6 +366,12 @@ def merge_persons(person_from, person_to):
     person_from.delete()
 
 
+class WrongWorkshopURL(ValueError):
+    def __str__(self):
+        return ('Event\'s URL doesn\'t match Github website format '
+                '"http://user.github.com/2015-12-08-workshop".')
```

Comment:
This error is raised only if we fall back to the old method, which requires GitHub website URL, and only if we don't get that URL format.